crates/edr_test_utils/src/lib.rs

```diff
@@ -1,47 +1 @@
 pub mod env;
-
```

Comment:
I move these into the `edr_provider` integration tests as they're only used there now.